modules/samples/s1/dashboards/pubsub-sample.json

```diff
@@ -55,24 +82,18 @@
                   "url": "store://gadget/subscriber/index.xml"
                 },
                 "listen": {
-                  "state-selected": {
-                    "type": "address",
-                    "description": "Used to filter based on state",
+                  "subscriber": {
+                    "type": "message",
+                    "description": "Used to listen to Any message of type address",
```

Comment:
The description contains uppercase character in the middle where is it should be all lower case characters as it's one sentence.